src/gui/test/screen_test_selftest.cpp

```diff
@@ -8,19 +8,20 @@
 #include "DialogHandler.hpp"
 #include "ScreenShot.hpp"
 #include "selftest_state_names.hpp"
+#include "selftest_result_type.hpp"
 
 #include <dirent.h>
 
 static void WaitLoop(uint32_t switch_period = 1024) {
-    uint32_t switch_time = gui::GetTick();
-    while ((gui::GetTick() - switch_time) < switch_period) {
+    uint32_t switch_time = gui::GetTick_IgnoreTickLoop();
+    while ((gui::GetTick_IgnoreTickLoop() - switch_time) < switch_period) {
         gui::TickLoop();
         gui_loop();
         DialogHandler::Access().Loop();
     }
 }
 
-static void WaitShotWait(const char *file_name, uint32_t wait = 512) {
+static void WaitAndShot(const char *file_name, uint32_t wait = 512) {
     char buff[128];
```

Comment:
Do we really need 128 bytes? What is the longest filename we'll process? Think about stack usage :wink:

Reply:
I have plenty of stack. It is debug only feature. Printer does not do anything else.
Ok will limit it to 64 to make you happy